### ISSUE TITLE
Metrics Stage 4: Create reports of metrics data

### ISSUE THREAD

**Issue by john-thomas-dotcms** [dotCMS/core#26955]
We need to create reports, using a 3rd party reporting tool, which aggregate and display the metrics data.

Step 1: Identify potential tools.
Step 2: Create a POC of one or more tools.
Step 3: Create a report for business use.

**Comment by john-thomas-dotcms** [comment]
We're going to use Grafana, since engineering already uses it, and it has a connector for Clickhouse.

Current Progress:
* I have access to our Grafana instance
* @spbolton already created a connector in Grafana (using Prometheus rather than direct to Clickhouse).
* @manuel-miranda helped me add the ClickHouse plugin to Grafana, so we can set up a direct connection later if needed.

Current Blockers:
1. We need to separate out the metrics data into a different table or database before we can properly set up the reports.
    * https://github.com/dotCMS/core/issues/27130
2. The existing data source in Grafana isn't connecting to the analytics infrastructure.
    * Needs troubleshooting, and all 3 of the best people to do that work are on PTO until next week.

**Comment by john-thomas-dotcms** [comment]
First pass of this is complete.  We'll need another card to migrate the first-pass reports to a new tool.

But that likely won't happen until we complete some planned hires, so we'll put in new cards when we have the resources to implement them.
